client/app/components/routes/select-operator/form/UserOperator.tsx

```diff
@@ -104,11 +105,22 @@ export default async function UserOperator({
 
   return params?.id === "request-access" ? (
     // If the operator exists then show the form from the second page
-    <UserOperatorContactForm formData={formData} schema={userOperatorPage2} />
+    <>
+      <UserOperatorContactForm formData={formData} schema={userOperatorPage2} />
+    </>
   ) : (
-    <UserOperatorMultiStepForm
-      schema={createUserOperatorSchema(businessStructuresList)}
-      formData={formData}
-    />
+    <>
+      <Note
+        classNames={"mb-4 mt-6"}
+        showNotePrefix={false}
+        showAlertIcon={true}
+        message="Please fill out missing information or update incorrect information about your operator to continue with your application.
+Some fields cannot be edited. If you need to change those fields, please contact us via email at GHGRegulator@gov.bc.ca."
+      />
```

Comment:
Should we display this component only when the user needs to update an existing operator?

Reply:
That makes sense, yes. I ~thought that was what I was doing here within the existing conditionality--is it not looking right for you?

Reply:
It seems that the Note component is being generated even for new operators, which may not be the intended behavior. Please correct me if I'm wrong.